cross-dom-comm/README.md

```diff
@@ -557,8 +557,8 @@ Unless we are going to run this code thousands of times, it is more efficient to
       
 ```
 
-There are two possibilities for the cross domain messenger's address on L1, because the address is not determined by Optimism.
-On L2 Optimism has full control of the genesis block, so we can put all of our contracts on convenient addresses.
+There are two possibilities for the cross domain messenger's address on L1, because the address is not determined by The Optimism Foundation.
```

Comment:
Doesn't need to be changed, but noting in some spots we've used 'the Optimism Foundation' and in some 'The Optimism Foundation'. Once this round of changes are done let's standardize on one of them. For now I'll capitalize when I see it

Reply:
Noted (https://linear.app/optimism/issue/DEVRL-1004/use-the-optimism-foundation-as-the-standard-capitalization)